lib/Runtime/Language/InterpreterLoop.inl

```diff
@@ -232,15 +243,27 @@ SWAP_BP_FOR_OPCODE:
                 break;
             }
             case INTERPRETER_OPCODE::ExtendedMediumLayoutPrefix:
+            case INTERPRETER_OPCODE::DblExtendedMediumLayoutPrefix:
             {
 #ifndef INTERPRETER_ASMJS  // Asmjs doesn't have any extended opcodes for now, remove that case
```

Comment:
Yes, and WASM. I think we just never ran anything with over 256 SIMD bytecode regs. Will fix this separately against release/1.2.